desktop-exporter/app/types/api-types.ts

```diff
@@ -0,0 +1,66 @@
+export type TraceSummary = {
```

Comment:
This is a direct translation of the types in https://github.com/CtrlSpice/otel-desktop-viewer/blob/main/desktop-exporter/types.go